apps/plea/models.py

```diff
@@ -452,30 +452,58 @@ def has_court(self, urn):
         """
         Take a URN and return True if the region_code is valid
         """
+        return self.filter(region_code=urn[:2],
+                           enabled=True).exists()
 
-        try:
-            self.get(region_code=urn[:2],
-                     enabled=True)
+    def get_by_urn(self, urn):
+        """
+        Retrieve court model by URN
+        """
 
-            return True
+        courts = self.filter(region_code=urn[:2],
```

Comment:
No, it's fine in this instance. Previously Court.region_code was a unique field, and this PR removes the unique index, but it's still OK if multiple courts have that region_code.